packages/codemods/src/codemods/v2.x.x/updateResolverTypes/updateResolverTypes.yargs.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+
+import fg from 'fast-glob'
+import task, { TaskInnerAPI } from 'tasuku'
+
+import getRWPaths from '../../../lib/getRWPaths'
+import runTransform from '../../../lib/runTransform'
+
+export const command = 'update-resolver-types'
+export const description =
+  '(v2.x.x->v2.x.x) Wraps types for "relation" resolvers in the bottom of service files'
```

Comment:
I always get confused by the versioning of these codemods. But isn't it ->v3 here?

Reply:
Yeah you're right - autogen strikes again!